doc/using-js-ipfs-in-the-browser.md

```diff
@@ -0,0 +1,70 @@
+Using JS IPFS in The Browser
+----------------------------
+
+The document describes how you can use JS IPFS in the browser. JS IPFS is the
+implementation of the IPFS protocal for browser. JS IPFS runs in a browser as a
+service worked, a web extenstion and node.js. The document describes how you can
+boost the speed and reduce the errors you face often because of limitation of
+js-ipfs.
+
+There are few different components for implementing JS IPFS in the browser. These
+are webrtc and websocket-star which will make it easier for implemented JS IPFS
+in the browser and boost the speed and reduce the errors.
+
+You can find the information present for enabling webRTC support for js-ipfs in
+the browser [here](https://github.com/ipfs/js-ipfs#how-to-enable-webrtc-support-for-js-ipfs-in-the-browser) and a different one is
+implementing with webrtc-star, you can find information [here](https://github.com/ipfs/js-ipfs#is-there-a-more-stable-alternative-to-webrtc-star-that-offers-a-similar-functionality).
+
+You can find information about running IPFS in the broswer [here](https://github.com/ipfs/js-ipfs#table-of-contents).
+
+Getting Data From IPFS
+-----------------------
+
+Using Javascript in the browser
+
+```js
+const node = new IPFS()
+
+node.once('ready', () => {
+  node.pipe('QmPChd2hVbrJ6bfo3WBcTW4iZnpHm8TEzWkLHmLpXhF68A', (err, data) => {
+    if (err) return console.error(err)
+
+    // convert Buffer back to string
+    console.log(data.toString())
+  })
+})
+```
+
+JS IPFS Restriction in Web Browser
+------------------------------------------
+
+- There is no DHT in js-ipfs. 
```

Comment:
This is no longer accurate. There is a DHT just not enabled by default. This PR is working on integrating it by default https://github.com/ipfs/js-ipfs/pull/1994